### PR TITLE
[1329][IMP] prepare 14.0

### PR DESCRIPTION
**Pull request by yostashiro**
Task [1329](https://www.quartile.co/web?debug#id=1329&action=771&active_id=449&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,93 +1,97 @@
...
+
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM=1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb
+ARG ODOO_SOURCE=odoo/odoo
+ARG ODOO_VERSION=14.0
+
+# Generate locale C.UTF-8 for postgres and general locale data
+ENV LANG C.UTF-8
+
+COPY --from=odoo-code odoo /opt/odoo
```
Comment (yostashiro) [inline Dockerfile:22]: As I check the image size again, I am not sure if the multi-stage build is helping here. Better double check.

**Comment by TimLai125** [comment]
@yostashiro The PR is ready to be reviewed again

File: Dockerfile [hunk 2]
```diff
@@ -1,93 +1,98 @@
...
-    libxtst6 \
-    fontconfig \
-    xfonts-75dpi \
-    xfonts-base \
-    libpng12-0 \
-    libjpeg-turbo8 \
-  && dpkg -i /opt/sources/wkhtmltox.deb \
-  && rm -rf /opt/sources/wkhtmltox.deb
+# Clean up apt cache
+RUN rm -rf /var/lib/apt/lists/* && apt autoremove -yqq
```
Comment (yostashiro) [inline Dockerfile:73]: Isn't this line redundant?

File: Dockerfile [hunk 3]
```diff
@@ -1,93 +1,98 @@
-FROM ubuntu:18.04
-MAINTAINER Quartile Limited <info@quartile.co>
+FROM debian:buster-slim
```
Comment (yostashiro) [inline Dockerfile:1]: Wouldn't it be better to use `python:3.8-slim-buster`? Debian 10 comes with Python 3.7 which I reckon will be kind of "the least tested" version among the Odoo community.

**Review by yostashiro — COMMENTED** [review]
@TimLai125 Can you please mention credits to the authors of the files we looked at in README?

File: Dockerfile (hunk 3 above)
Comment (TimLai125) [reply Dockerfile:1]: I tried to use `python:3.8-slim-buster` and compare the size which is 500 mb bigger. Therefore I choose to use `debian:buster-slim` in the end.

Comment (yostashiro) [reply Dockerfile:1]: What's your idea about the Python version, though? OCA will not be testing with Python 3.7, so I don't feel very safe with this version.

**Comment by TimLai125** [comment]
Merging the PR for now to test

**Pull request merged**